model/Licensing/Properties/obsoletedBy.md

```diff
@@ -0,0 +1,25 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# obsoletedBy
+
+## Summary
+
+Specifies the licenseId that is preferred to be used in place of a deprecated
+License or LicenseAddition.
+
+## Description
+
+An obsoletedBy value for a deprecated License or LicenseAddition specifies
+the licenseId of the replacement License or LicenseAddition that is preferred
+to be used in its place. It should use the same format as specified for a
+licenseId.
+
+The License's or LicenseAddition's comment value may include more information
+about the reason why the licenseId specified in the obsoletedBy value is
+preferred.
+
+## Metadata
+
+- name: obsoletedBy
+- Nature: DataProperty
+- Range: xsd:string
```

Comment:
It's possible it could be a license expression rather than a single license. In the [license-list-XML schema](https://github.com/spdx/license-list-XML/blob/main/schema/ListedLicense.xsd) I believe it's essentially a string, so I'd be inclined to leave it as that for the moment.